Yubico.YubiKey/docs/users-manual/application-piv/pin-puk-mgmt-key.md

```diff
@@ -32,11 +32,17 @@ is a 6- to 8-byte value, each of the bytes an ASCII number ('0' to '9', which in
 (upper- and lower-case), and even non-alphanumeric characters such as !, %, or # (among
 others).
 
-The PUK is used to unblock the PIN (see the section below on Blocking). The standard
-specifies that it is to be an 8-byte value, each of the bytes any binary value (`0x00` -
-`0xFF`). If your application uses the keyboard to insert the PUK, you might limit the user
-to ASCII characters, but the YubiKey will accept any byte value in the PUK. In addition,
-the YubiKey will allow the PUK to be 6, 7, or 8 bytes long.
+The PUK is used to unblock the PIN (see the section below on [blocking](#blocking)). The standard
+specifies that the PUK is to be an 8-byte value, with each of the bytes any binary value from `0x00` to
+`0xFF`. The YubiKey, however, will allow the PUK to be 6, 7, or 8 bytes long. For YubiKeys with firmware versions prior to 5.7, the key will accept any value in the `0x00` -
```

Comment:
```suggestion
`0xFF`. For YubiKeys with firmware versions prior to 5.7, the key will accept any value in the `0x00` -
```

Reply:
The PUK is always 8 characters, the PIN can be 6, 7 or 8 characters.

Reply:
The PUK can actually be 6-8 characters too. Just confirmed the behavior with the authenticator app.